src/shared/molecules/DataExplorerGraphFlowMolecules/NavigationArrow.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ArrowLeftOutlined, ArrowRightOutlined } from '@ant-design/icons';
+import './styles.less';
+
+type NavigationArrowDirection = 'back' | 'forward';
+
+const NavigationArrow = ({
+  direction,
+  visible,
+  title,
+  onClick,
+}: {
+  direction: NavigationArrowDirection;
+  visible: boolean;
+  title: string;
+  onClick: () => void;
+}) => {
+  return visible ? (
+    <button
+      className={`navigation-arrow-btn ${direction}`}
+      aria-label={`${direction}-arrow`}
```

Comment:
I think the `title` prop is not really needed. It's just `direction`, and we can add `text-transform: capitalize;` to it.

Reply:
Generally, verb prop must be separated from display prop, so you are separating the `logic` from `presentation`
and later if you want to change the title will not cause any code modification.